core/src/main/scala/org/apache/spark/rdd/UnionRDD.scala

```diff
@@ -18,10 +18,10 @@
 package org.apache.spark.rdd
 
 import java.io.{IOException, ObjectOutputStream}
+import java.util.concurrent.ForkJoinPool
 
 import scala.collection.mutable.ArrayBuffer
 import scala.collection.parallel.ForkJoinTaskSupport
-import scala.concurrent.forkjoin.ForkJoinPool
 import scala.reflect.ClassTag
```

Comment:
We cannot use `ThreadUtils.newForkJoinPool` in this class instead of `ForkJoinPool` directly?